utility/firebase.js

```diff
@@ -1,22 +1,23 @@
 /* eslint-disable no-console */
-import firebase from 'firebase/app'
+import firebase, { registerVersion } from 'firebase/app'
 import 'firebase/firestore'
 import 'firebase/auth'
 import 'firebase/storage'
 import * as Parser from 'json2csv'
 
 if (!firebase.apps.length) {
   const config = {
-    apiKey: process.env.FIREBASE_API_KEY,
-    authDomain: process.env.FIREBASE_AUTH_DOMAIN,
-    databaseURL: process.env.FIREBASE_DATABASE_URL,
-    projectId: process.env.FIREBASE_PROJECT_ID,
-    measurementId: process.env.FIREBASE_STORAGE_BUCKET,
-    appId: process.env.FIREBASE_APP_ID,
-    storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
-    messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID
+    apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
+    authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
+    databaseURL: process.env.NEXT_PUBLIC_FIREBASE_DATABASE_URL,
+    projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+    measurementId: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
+    appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
+    storageBucket: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
+    messagingSenderId: process.env.NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID
   }
   firebase.initializeApp(config)
+  console.log(config.apiKey)
```

Comment:
Should we remove this console.log here for security?

Reply:
oops lol forgot to delete that